### ISSUE TITLE
color code followed by reset style results in bad output

### ISSUE THREAD

**Issue by bandophahita** [jenkinsci/ansicolor-plugin#172]
The following ansi code combo -- light green with normal style (reset sgr) -- appears to work ok.
`sh "echo -en '\\033[92;22mlightgreen reset sgr \\033[0m\n';"`
![image](https://user-images.githubusercontent.com/4211287/76109169-8bea8f80-5fa1-11ea-9881-38b3f4d8d282.png)

However, when the ansi codes are separated the output appears to break
`sh "echo -en '\\033[92m\\033[22mlightgreen reset sgr \\033[0m\n'"`
![image](https://user-images.githubusercontent.com/4211287/76109275-c3593c00-5fa1-11ea-8ed6-8f3928f3df51.png)

Expected behavior on the second example should match that of the first.

It's worth noting that if any SGR code is set prior to the `\\033[22m` being called, the output works fine.
`sh "echo -en '\\033[92m\\033[1mlightgreen bold \\033[92m\\033[22mlightgreen normal\\033[0m\n';"`
![image](https://user-images.githubusercontent.com/4211287/76109743-b7ba4500-5fa2-11ea-8afb-ce3dcf33487b.png)

The problem appears to be when `22m` is called and no SGR style has been set the output breaks -- but again only when the `22m` is not in a combined ansi code.  
(e.g.`\\033[92m\\033[22m` **bad** vs  `\\033[92;22m` **ok**

Using ansicolor 0.6.2 currently on Jenkins 2.204.1 in a scripted pipeline.

### Test environment

- Jenkins Version: 2.204.1 
- AnsiColor Version: 0.6.2 
- Hostsystem: internal network (no outside access)

### Steps to reproduce the behavior

jenkinsfile scripted example used in regular pipeline. 
```Groovy
node {
    stage('color'){
        ansiColor('xterm'){
            // breaks
            sh "echo -e '\\033[92m\\033[22mlightgreen and separate reset sgr \\033[0m'"
            // works ok
            sh "echo -e '\\033[92;22mlightgreen and combined reset sgr \\033[0m'"
            // works ok
            sh "echo -en '\\033[92m\\033[1mlightgreen bold \\033[92m\\033[22mlightgreen normal\\033[0m'"
        }
    }
}
```

**Comment by tszmytka** [comment]
Thank you for a terrific issue report. I can confirm it is reproducible also in the newest version `0.6.3` so it is definitely a bug.

I've added a test case based on your input. Solution should not be very complicated. SGR Normal `ESC[22m` doesn't seem to be very popular and its implementation has a lot of potential for improvement.

**Comment by okovpashko** [comment]
It seems like `ESC[22m` is pretty popular in Node.js packages. 

Finally figured out how to force colors in Jenkins console and got the same issue with displaying the escape sequences in almost all the colorized output:
---
![image](https://user-images.githubusercontent.com/6490265/79147476-a7309200-7dcc-11ea-953b-dd89f00c9580.png)
---
![image](https://user-images.githubusercontent.com/6490265/79147442-9c75fd00-7dcc-11ea-8d24-d968c0038c9a.png)
---
![image](https://user-images.githubusercontent.com/6490265/79147502-b4e61780-7dcc-11ea-8089-702281be9388.png)
---
![image](https://user-images.githubusercontent.com/6490265/79147532-c16a7000-7dcc-11ea-9de1-6d3f7a4b7f31.png)
---

Looking forward for the new plugin version with the fix.

**Comment by tszmytka** [comment]
Fixed and released in [v0.7.0](https://github.com/jenkinsci/ansicolor-plugin/releases/tag/ansicolor-0.7.0)
